src/pages/console/auth/import.mdx

```diff
@@ -36,7 +36,7 @@ We can also use Amplify CLI to [import a Cognito user pool resource](/cli/auth/i
 
 ## Import an existing identity pool
 
-![import Amazon Cognito resource](/images/studio/import-identity-pool.png)
+![Process to import Amazon Cognito identity pool resource](/images/studio/import-identity-pool.png)
```

Comment:
```suggestion
![Process to import an Amazon Cognito identity pool resource](/images/studio/import-identity-pool.png)
```

Reply:
I think this image could be decorative. I'm not sure what info I'm supposed to get out of this pic other than we're on step 3 and need to select an Identity Pool. 

I might just be missing something, but is there something we're trying to convey with the pic @ykethan?

Reply:
@timngyn yes, so on studio for imports can enable to disable importing a identity using a toggle. but by default this is on we can disable this.